src/selfplay/tournament.cc

```diff
@@ -209,10 +218,10 @@ void SelfPlayTournament::PlayOneGame(int game_number) {
     Mutex::Lock lock(mutex_);
     player1_black = ((game_number % 2) == 1) != first_game_black_;
     if (!openings_.empty()) {
-      if (player_options_[0].Get<bool>(kOpeningsMirroredId)) {
+      if (player_options_[0][!player1_black].Get<bool>(kOpeningsMirroredId)) {
```

Comment:
I don't think these openings flags make sense to try and lookup a specific sub dictionary.  Much like below where you did [0][0] for an openings flag, I think the same could be done for this one, and for the 'random' check immediately below.

Reply:
Done.